keps/sig-release/2853-kubernetes-kubernetes-repository-branch-rename/README.md

```diff
@@ -0,0 +1,769 @@
+-->
+
+Many communities, both on GitHub and in the wider Git community, are considering renaming the default branch name of their repository from `master`. The Kubernetes GitHub repositories is gradually renaming the default branch of our own repositories from `master` to `main`.
+This KEP is to we rename the branch for the main repository in the Kubernetes community.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+In the Kubernetes community and other communities, there is an effort to make all the language and terms more inclusive. One small step to make that is to change the current branch name to one that is more inclusive and does not harmful and exclusionary.
+
+Our biggest repository is [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes), and this has a lot of open Pull Requests and several other forks, clones that use that for downstream signal or a custom build.
+
+Changing the branch name might cause issues for downstream consumers, our contributors, and the community in general.
+
+For that, we are creating this KEP to have a plan to address the issues and communication.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+The main goal is to be more inclusive using better naming.
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+This applies only to the main Kubernetes GitHub repository [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes).
+Other repositories will follow a similar approach but at their timeframe.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+- We aim to make the change during the start of v1.25 release (spring 2022).
+- Perform a Survey to gather information on downstream consumers and how this might affects their workflow. See [Survey Questions](#survey-questions).
+- Announce the change in the kubernetes-dev and kubernetes-announcement mailing list and in a blog post. As well as use Twitter and other social media to announce.
+- Change all open Pull Requests to the `Draft mode` with that when we rename the branch, the tests will not be triggered and will avoid a massive queue in the CI infrastructure.
```

Comment:
```suggestion
- Change all open Pull Requests to `Draft mode` (so that when we rename the branch, tests will not be triggered, and we avoid a massive queue in the CI infrastructure).
```